steps/swift-package-manager-test-for-mac/0.0.1/step.yml

```diff
@@ -0,0 +1,62 @@
+title: Swift Package Manager Test for Mac
+summary: |
+  Runs Test on your macOS project with Swift Package Manager
+description: |
+  Runs Swift Package Manager's `swift test` action for macOS app projects.
+website: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac
+source_code_url: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac
+support_url: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac/issues
+published_at: 2019-03-10T16:07:57.802577+09:00
+source:
+  git: https://github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac.git
+  commit: b7a1e866f8af3402c6a4c9cd9285b4a31b923be6
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- macos
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/kitasuke/bitrise-step-swift-package-manager-test-for-mac
+deps:
+  brew:
+  - name: go
+  check_only:
+  - name: xcode
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- build_path: ./.build
+  opts:
+    description: |
+      A build path, relative to the working directory.
+    is_expand: true
+    is_required: false
+    title: Build Path
+    value_options: []
+- is_skip_build: "no"
+  opts:
+    description: |
+      Should skip building the test target?
+    title: Skip build
+    value_options:
+    - "yes"
+    - "no"
+- is_parallel: "no"
+  opts:
+    description: |
+      Should run the tests in parallel?
+    title: Parallel test
+    value_options:
+    - "yes"
+    - "no"
+outputs:
+- BITRISE_SWIFT_TEST_RESULT: null
```

Comment:
What is the purpose of this output?
Description suggests that the only purpose of this step is performing tests, so if they fail then step execution will just fail. No need for additional variable.

Reply:
That's absolutely right. Will fix it

Reply:
Addressed your feedback